src/app/_variables.scss

```diff
@@ -67,6 +59,21 @@ $easeInCubic: cubic-bezier(0.55, 0.055, 0.675, 0.19);
 $easeOutCubic: cubic-bezier(0.215, 0.61, 0.355, 1);
 $easeInOutCubic: cubic-bezier(0.645, 0.045, 0.355, 1);
 
+// Theme style properties that remain consistent across all themes
+
+// Expose primary brand colour as a css variable so it can be used in themes
+:root {
+  --dim-brand: #{$dim-brand};
+}
+
+// Tooltips and arrows
+$theme-tooltip-arrow-size: 8px; // Ensure this stays in sync with 'popperArrowSize' in 'usePopper.ts'
+$theme-tooltip-arrow-size-mini: 6px;
+$theme-tooltip-corner-radius: 6px;
+
+// Search
+$theme-corner-radius-search: 6px;
```

Comment:
> There's still a fair bit of work needed to make a light theme possible (e.g. updating foreground text colours, converting the numerous shades of gray to semi-transparent black/white etc.), but I assume that'll happen in a separate PR.

Yep, exactly.

There are also some visual/interaction tweaks i'd like to propose that would make sense ahead of introducing themes (things like interaction and decoration styles) so that the base css remains consistent across themes — but will also raise them as separate PRs